api/net/Areas/Editor/Models/Content/ItemModel.cs

```diff
@@ -0,0 +1,61 @@
+using MimeTypes;
+
+namespace TNO.API.Areas.Editor.Models.Content;
+
+/// <summary>
+/// ItemModel class, provides a model that represents an file or directory.
+/// </summary>
+public class ItemModel
```

Comment:
This model is a duplicate of the `ItemModel` for the `StorageController`, but you are not using it as a model.  You are only using for the functionality to extract some of the properties for the stream.  Don't duplicate code if you don't need to.  There are a few exceptions of this rule, which a true model would be one of them, but you are not using this model for the purpose of a model.

Reply:
This was duplicated for the same reason as ContentConfig.cs. I can use the functionality in ItemModel under storage if that's the best approach.

Reply:
Waiting for your response before making changes and resubmitting.

Reply:
Just use the `ItemModel` that already exists if you need the same functionality and you're not actually using the model for what a model is used for.  Models are for serialization.  We duplicate models because they are just properties with no logic.  That way endpoints can mutate without breaking other endpoints.

Reply:
Altered the code to use the ItemModel.cs in Storage.